unitex/src/fr/umlv/unitex/frames/ConjugaisonFrame.java

```diff
@@ -0,0 +1,713 @@
+import javax.swing.JMenu;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTextField;
+import javax.swing.border.EmptyBorder;
+import javax.swing.filechooser.FileFilter;
+
+
+import fr.umlv.unitex.config.Config;
+import fr.umlv.unitex.config.ConfigManager;
+import fr.umlv.unitex.config.PreferencesListener;
+import fr.umlv.unitex.config.PreferencesManager;
+import fr.umlv.unitex.files.FileUtil;
+import fr.umlv.unitex.process.Launcher;
+import fr.umlv.unitex.process.ToDo;
+import fr.umlv.unitex.process.commands.MultiFlexCommand;
+import fr.umlv.unitex.text.BigTextList;
+
+public class ConjugaisonFrame extends JInternalFrame {
+	
+	private static final int MAIN_PANEL_WIDTH = 450;
+	private static final int MAIN_PANEL_HEIGHT = 400;
+	private static final String DOT_SEPARATOR = "\\.";
+	private static final String PLUS_SEPARATOR = "\\+";
+	private static final String COMMMA_SEPARATOR = ",";
+	private static final String DOUBLESLASH_SEPARATOR = "/ /";
+	private static final String ENCODE = "utf-8";
+	private static final String ENCODE_UTF_16 = "utf-16le";
+//	private static final String UTF8_BOM = "\uFEFF";
+	private static final List<String> exCludeChar = new ArrayList<String>(); 
+	private static String resultFileName = null;
+	private static final List<String> graphNameList = new ArrayList<String>(); 
+	
+	private static final String folderReferenceName = "Conjugaison";
+	private static final String resultPath = Config.getUserCurrentLanguageDir()+File.separator+folderReferenceName+File.separator+"result.txt";
+	private static final String conjugaisonFolderPath = Config.getUserCurrentLanguageDir()+File.separator+folderReferenceName;
+	private static final String inflexionFolderPath = conjugaisonFolderPath+File.separator+"inflexion";
+	private static final String exploreGrapheFolderPath = conjugaisonFolderPath+File.separator+"exploregraphe";
+	private static final String resultFolderPath = conjugaisonFolderPath+File.separator+"result";
+	
+	private final JTextField inputText = new JTextField();
+	private final JButton btnGenerateAgglutinedFormsFromFile = new JButton("Generate agglutinated forms");
+	private final JButton setGraphFile = new JButton("Set...");
+	private final JTextField graphFilePath = new JTextField();
+	private final BigTextList textArea = new BigTextList();
+	private JComboBox graphList = null;
+	
+	private static final String ALL = "All";
+	private static final String placeHolderLemma = "$V.LEMMA$";
+	private static final String placeHolderCode = "$V.CODE$";
+	private static final String placeHolderVerb = "$V$";
+	private static final String tmpSuffix ="tmp.dic";
+	private static final String flxSuffix ="flx.dic";
+	private static final String tmpFlxSuffix ="tmpflx.dic";
+	private static final String ERROR_FILE_EMPTY = "Please choose a dictionary file";
+	private static final String ERROR_RESULT_EMPTY = "Current criteria has not find result";
```

Comment:
"has not find" does not seem to be grammatically correct. What about something like:  `No results were found using the criteria provided`